CHANGELOG.rst

```diff
@@ -38,8 +38,7 @@ Version 2021-dev
 
 Version 1.6.4 (released XX.12.20)
 =================================
-
--  no changes
+-  create script to convert help to rst (#312)
```

Comment:
Nope, this needs under the Version 2021-dev section unless we merge it into `stable`, which I wouldn't do at this point.